<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-GORURFBI" ignored="1" name="Default test(failed)(failed)" tests="2" failures="2" timestamp="29 Jan 2021 10:31:39 GMT" time="0.033" errors="0">
  <testcase name="tc2" time="0.009" classname="org.login.tester.A1">
    <failure type="java.lang.AssertionError" message="check the fauiler expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: check the fauiler expected [true] but found [false]
at org.login.tester.A1.tc2(A1.java:15)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- tc2 -->
  <testcase name="tc3" time="0.001" classname="org.login.tester.A1">
    <failure type="java.lang.AssertionError" message="check the fauiler expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: check the fauiler expected [true] but found [false]
at org.login.tester.A1.tc3(A1.java:21)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- tc3 -->
  <testcase name="tc1" time="0.0" classname="org.login.tester.A1">
    <ignored/>
  </testcase> <!-- tc1 -->
</testsuite> <!-- Default test(failed)(failed) -->
